### PR TITLE
Feature/dark theme

### PR DESCRIPTION
**Pull request by emonnin-epfl**

### CODE REVIEW

**Review by ozan-guven — CHANGES_REQUESTED** [review]
In general, good code ! But try those changes.

File: app/src/main/java/ch/epfl/sdp/healthplay/auth/SignedInActivity.java [hunk 1]
```diff
@@ -90,6 +93,24 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
     }
 
+    public void onClickNight(View view) {
```
[the remark this answers is not in the document]
Comment (emonnin-epfl) [reply app/src/main/java/ch/epfl/sdp/healthplay/auth/SignedInActivity.java:96]: fixed!

File: app/src/main/java/ch/epfl/sdp/healthplay/Frag_Home.java [hunk 2]
```diff
@@ -180,7 +180,7 @@ public void onDataChange(@NonNull DataSnapshot snapshot) {
                         userStats.get(date).put(Database.WEIGHT, String.valueOf(value.get(Database.WEIGHT)));
 
                         //print the changes only if they happened on the focused date
-                        if (selected_Date.equals(date)) {
+                        if (selected_Date != null && selected_Date.equals(date)) {
```
[the remark this answers is not in the document]
Comment (emonnin-epfl) [reply app/src/main/java/ch/epfl/sdp/healthplay/Frag_Home.java]: did it!

File: app/src/main/java/ch/epfl/sdp/healthplay/auth/ProfileActivity.java [hunk 3]
```diff
@@ -84,9 +89,11 @@ public void initBirthday(String userId) {
             @Override
             public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                 String birthday = dataSnapshot.getValue(String.class);
-                String[] parts = birthday.split("-");
-                birthday = parts[2] + "/" + parts[1] +"/" + parts[0];
-                TextViewBirthday.setText(birthday);
+                if(birthday != null) {
+                    String[] parts = birthday.split("-");
```
[the remark this answers is not in the document]
Comment (emonnin-epfl) [reply app/src/main/java/ch/epfl/sdp/healthplay/auth/ProfileActivity.java:93]: good observation, fixed it !

File: app/src/main/java/ch/epfl/sdp/healthplay/Frag_Home.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (ozan-guven) [reply app/src/main/java/ch/epfl/sdp/healthplay/Frag_Home.java]: Perfect !

**Review by ozan-guven — APPROVED** [review]
Perfect ! Your code is much better and I cannot find any other things to say. Good job!

**Review by Antonio06188 — CHANGES_REQUESTED** [review]
In many activity, in onCreate method you write the same code, you can maybe make a static method

**Comment by emonnin-epfl** [comment]
did it!

**Review by Antonio06188 — APPROVED** [review]

**Pull request merged**